atm.rb

```diff
@@ -0,0 +1,534 @@
+require 'csv'
+
+module Validators
+  def self.number?(num)
+    if num.match(/\d+/)
+      return true
+    else
+      return false
+    end
+  end
+  def self.get_number(mess)
+    puts mess
+    num=gets
+    if !number?(num)
+      puts "Must be Integer.."
```

Comment:
this function should only return num, printing should be handled where we are calling this method